### ISSUE TITLE
Is it possible to share the code of the regression baseline?

### ISSUE THREAD

**Issue by ZezhouCheng** [jasonyzhang/relpose#3]
Thanks for sharing the code again!

I'm playing with some baselines of relative pose estimation. Is it possible to share the implementation of the direct regression baseline? In my experiments, it seems necessary to have batch norm layers after each of the last linear layers, otherwise, the training loss does not decrease. Based on Figure 17, it seems there is no batch norm layer in the relative pose predictor. I'm curious about the exact settings of the baselines to see if there are some details I have missed. It is totally fine if the codes have not yet been cleaned up; I just wanna see if I have missed some implementation details.

**Comment by jasonyzhang** [comment]
Hi,

I attached the code for the direct regression model. I did not use batch norm between layers, but I still found that the loss decreased and trained it until convergence.

[models.txt](https://github.com/jasonyzhang/relpose/files/11629712/models.txt)
